pvlib/pvsystem.py

```diff
@@ -1149,6 +1166,9 @@ class Array:
         Valid strings are 'open_rack', 'close_mount', and 'insulated_back'.
         Used to identify a parameter set for the SAPM cell temperature model.
 
+    array_losses_parameters: None, dict or Series, default None.
```

Comment:
sorry, should have caught this earlier.... it seems repetitive to call this parameter `array_losses_parameters` since it's already on an `Array`. I suppose there is some risk of user confusion if this and the `PVSystem` `losses_parameters` are called the same thing, but it doesn't hurt to have extra keys in these dictionaries. It's unlikely that would change in the future.

Maybe we should live with the current API on master in an alpha release or two and then maybe take up renaming in a follow up issue. I could see something like #1176 providing more clarity on this.

Thoughts @ncroft-b4 @cwhanse ?

Reply:
I'm OK with either `array_losses_parameters` or `Array.losses_parameters` (and `PVSystem.losses_parameters`) in this PR.

I think as loss models are added to pvlib and we work those models into ModelChain, we'll end up refactoring the parameter containers.

Reply:
Yeah I think my concern was making it distinguishable from the `PVSystem` parameter. Also, to keep compatibility with initializing a `PVSystem` directly, there needs to be a difference between the `losses_parameters` that would be passed into the `Array` and the ones that would stay with the `PVSystem`. 

That makes me realize that I added `array_losses_parameters` to the list of keys that get passed to `Array` from `PVSystem` if an `Array` is not specified, but I did not add `array_losses_parameters` explicitly to the documentation for `PVSystem` or to the `init`. That makes sense to be because `array_losses_parameters` is only an `Array` parameter, but it does make it different from the other `Array` parameters that are still part of `PVSystem` by legacy. I'm good with it as it is but if you want to keep it uniform with the legacy parameters we could change it.